src/NuGetPackageVerifier/Rules/PackageOwnershipRule.cs

```diff
@@ -25,7 +25,7 @@ public class PackageOwnershipRule : IPackageVerifierRule
             "microsoft",
         };
         private static readonly IList<string> OwnedPackages = GetOwnedPackages();
-        private readonly HttpClient _httpClient = new HttpClient();
+        private static readonly HttpClient _httpClient = new HttpClient();
```

Comment:
`_httpClient` => `HttpClient` to keep it consistent with the other `static readonly`s